CHANGES.md

```diff
@@ -22,6 +22,8 @@
   is 4.08 to 4.13 now (#345, @Leonidas-from-XIV)
 - Do not install deprecated `mdx` binary anymore (#274, @gpetiot)
 - Remove deprecated `rule` command (#312, @gpetiot)
+- Remove support for `require-package` label, use the `mdx` stanza in dune
```

Comment:
Maybe mention here that it was only used by MDX rule and that it can otherwise be safely removed!